File: web/plugins/contains.js
import { WebPlugin } from "../engine/plugin.js";

export class ContainsPlugin extends WebPlugin {
    constructor() {
        super("文字が含まれる");
    }

    async execute(line, context) {
        const match = line.match(/^文字が含まれる\s+「(.+)」\s+「(.+)」$/);
        if (!match) return;

        context.runtime.write(
            match[1].includes(match[2])
        );
    }
}

File: pc/plugins/contains.py
import re
from core.plugin import Plugin


class ContainsPlugin(Plugin):
    def translate(self, line):
        match = re.fullmatch(r'文字が含まれる\s+「(.+)」\s+「(.+)」', line)
        if not match:
            return None

        text = match.group(1)
        search = match.group(2)

        return f"print({search!r} in {text!r})"